fluentd/out_syslog_buffered.rb

```diff
@@ -91,11 +91,27 @@ def write(chunk)
 
     def send_to_syslog(tag, time, record)
       tag = tag.sub(@remove_tag_prefix, '') if @remove_tag_prefix
-      @packet.hostname = hostname
       if @use_record
-        @packet.facility = record['facility'] || @facilty
-        @packet.severity = record['severity'] || @severity
+        @packet.hostname = record['hostname'] || hostname
```

Comment:
There is a bunch of this code that can be refactored into a module that could be included into both files.  We should consider doing that if time allows.

Reply:
I agree.  Could you please teach me how to make it happen?   Just creating a method in one class and "require" it from another did not work.  I was hoping to do so using macro but so far no luck...